buildstockbatch/gcp/gcp.py

```diff
@@ -5,11 +5,16 @@
 ~~~~~~~~~~~~~~~
 This class contains the object & methods that allow for usage of the library with GCP Batch.
 
-This implementation tries to match the structure of `../aws/aws.py` in the 'nrel/aws_batch' branch
-as much as possible in order to make it easier to refactor these two (or three, with Eagle) to share
-code later. Also, because that branch has not yet been merged, this will also _not_ do any
-refactoring right now to share code with that (to reduce merging complexity later). Instead, code
-that's likely to be refactored out will be commented with 'todo: aws-shared'.
+Architecture overview:
```

Comment:
I'm not sure what the right level of detail here is, but I felt we were missing a high-level outline of how we're using GCP (the kind that would have been useful when we were first trying to understand the AWS implementation).

Reply:
👍! I think you should add that many of these things are done by the base classes (including sampling, and input files).

I might suggest that a README.md would be better, especially since I think it'd be best to document more holistically (i.e., relationship with docker_base and aws), but I'm fine with this incremental improvement, at least.

Reply:
Added a note that this logic is split between the two places.

Yeah, a readme might be better if we want to add more detail later. I'm thinking of this section as mainly "Here's how we're using GCP", which is pretty specific to gcp.py, but a more general "Here's how the code is structured" could also be helpful.